rfcs/0011_Arm_Ethos-U_Integration.md

```diff
@@ -125,12 +127,9 @@ primfn(placeholder_1: handle, placeholder_2: handle, placeholder_3: handle, etho
 }
 ```
 
+### C4. Translating TIR Primfuncs to C-sources that call to the driver APIs to perform the execution.
 
-Given, that the complexity of this component, we'll be putting up a seperate RFC to describe the functionality of Ethos™-U TE/TIR Compiler in detail.
-
-### C4. Translating Ethos™-U TIR Primfuncs to C-sources that call to the Ethos™-U driver APIs to perform the execution.
-
-Ethos™-U hardware is used from the host CPU via invoking a driver API call with a command stream (a Ethos™-U specific binary artefact) that describes the hardware operators that need to execute. This component will use the TIR Primfunc to extract the hardware operators and buffer information. Thereafter, we'll be using Arm® Vela (https://pypi.org/project/ethos-u-vela/) compiler's backend python APIs to convert the TIR Primfunc to a command stream. Finally, the generated command stream will be wrapped in a c-source that invokes it using the Ethos™-U driver APIs.
+The hardware is used from the host CPU via invoking a driver API call with a command stream (a hardware specific binary artefact) that describes the hardware operators that need to execute. This component will use the TIR Primfunc to extract the hardware operators and buffer information. Thereafter, we'll be using Arm® Vela (https://pypi.org/project/ethos-u-vela/) compiler's backend python APIs to convert the TIR Primfunc to a command stream. Finally, the generated command stream will be wrapped in a c-source that invokes it using the driver APIs.
```

Comment:
More of a question about style, are we ok with mixing British and American spellings in documents?

Reply:
Fixed it to be consistent.